chromium_src/chrome/browser/extensions/chrome_component_extension_resource_manager.cc

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/extensions/chrome_component_extension_resource_manager.h"
+
+#include "brave/components/brave_extension/grit/brave_extension_generated_map.h"
+#include "brave/components/brave_extension/grit/brave_extension_resources_map.h"
+#include "brave/components/brave_rewards/browser/buildflags/buildflags.h"
+#include "brave/components/brave_webtorrent/browser/buildflags/buildflags.h"
+
+#if BUILDFLAG(ENABLE_BRAVE_WEBTORRENT)
+#include "brave/components/brave_webtorrent/grit/brave_webtorrent_generated_map.h"
+#include "brave/components/brave_webtorrent/grit/brave_webtorrent_resources_map.h"
+#endif
+
+#if BUILDFLAG(BRAVE_REWARDS_ENABLED)
+#include "brave/components/brave_rewards/resources/extension/grit/brave_rewards_extension_resources_map.h"
+#include "brave/components/brave_rewards/resources/extension/grit/brave_rewards_panel_generated_map.h"
+#endif
+
+#define BRAVE_CHROME_COMPONENT_EXTENSION_RESOURCE_MANAGER_DATA \
```

Comment:
I'm not following why we're adding a method for this, why not just patch in the actual AddComponentResourceEntries lines?

Reply:
Addressed by d38a3c205635e1e16e801437e5d35be0c3d6db70